owncloudApp/src/main/java/com/owncloud/android/presentation/common/DrawerViewModel.kt

```diff
@@ -43,9 +47,12 @@ class DrawerViewModel(
     private val removeAccountUseCase: RemoveAccountUseCase,
     private val getUserQuotasUseCase: GetUserQuotasUseCase,
     private val localStorageProvider: LocalStorageProvider,
-    private val coroutinesDispatcherProvider: CoroutinesDispatcherProvider
+    private val coroutinesDispatcherProvider: CoroutinesDispatcherProvider,
+    private val contextProvider: ContextProvider,
 ) : ViewModel() {
 
+    fun getFeedbackMail() = contextProvider.getString(R.string.mail_feedback)
+
```

Comment:
Don't add this before the class properties, let's keep an order

Reply:
Moved below getUsernameOfAccount()